Resources/public/js/views/ez-locationviewview.js

```diff
@@ -213,6 +217,21 @@ YUI.add('ez-locationviewview', function (Y) {
 
                     );
                 }
+            },
+
+            /**
+             * The language switcher view instance
+             *
+             * @attribute languageSwitcherView
+             * @type eZ.LanguageSwitcherView
+             */
+            languageSwitcherView: {
+                valueFn: function () {
+                    return new Y.eZ.LanguageSwitcherView({
+                        content: this.get('content'),
+                        location: this.get('location')
```

Comment:
you should add the parent view as a bubble target of the languageSwitcher so that if it fires a event, this event can be handled at a higher level (parent view, view service, application, ...). To do that, you can write something like:
```js
return new Y.eZ.LanguageSwitcherView({
    content: this.get('content'),
    location: this.get('location'),
    bubbleTargets: this,
});
```